tool/tsh/common/tsh.go

```diff
@@ -4289,13 +4289,16 @@ func printLoginInformation(cf *CLIConf, profile *client.ProfileStatus, profiles
 
 	if len(accessListsToReview) > 0 {
 		fmt.Printf("Access lists that need to be reviewed:\n")
+		// date time format layout
+		const YYYYMMDD = "2006-01-02"
 		for _, accessList := range accessListsToReview {
 			d := time.Until(accessList.Spec.Audit.NextAuditDate).Round(time.Minute)
+			nextAuditDate := accessList.Spec.Audit.NextAuditDate.Format(YYYYMMDD)
 			var msg string
 			if d > 0 {
-				msg = fmt.Sprintf("%v left to review", d.String())
+				msg = fmt.Sprintf("review date is %v", nextAuditDate)
 			} else {
-				msg = fmt.Sprintf("review was required %v ago", (-d).String())
+				msg = fmt.Sprintf("review was required on %v", nextAuditDate)
 			}
```

Comment:
Do we still need `d`? We could something like that:
```go
	nextAuditDate := accessList.Spec.Audit.NextAuditDate.Format(time.DateOnly)
	var msg string
	if accessList.Spec.Audit.NextAuditDate.After(time.Now()) {
		msg = fmt.Sprintf("review was required on %v", nextAuditDate)
	} else {
		msg = fmt.Sprintf("review date is %v", nextAuditDate)
	}
```

It will look a bit wonky around the target date and can say review was required on *current date for viewer*, don't know if it matters. We also could switch to showing dates only if `d` is a large value.

Reply:
thanks, updated.